admin/css/debug-toolbar/_theme-dark.scss

```diff
@@ -12,141 +12,156 @@
 // ========================================================================== */
 
 #debug-icon {
-	background-color: $t-dark;
-	@include box-shadow(0, 0, 4px, $m-gray);
-	a:active, a:link, a:visited {
-		color: $g-orange;
-	}
+    background-color: $t-dark;
+    @include box-shadow(0, 0, 4px, $m-gray);
+
+    a:active,
+    a:link,
+    a:visited {
+        color: $g-orange;
+    }
 }
 
 
 // DEBUG BAR
 // ========================================================================== */
 
 #debug-bar {
- 	background-color: $t-dark;
- 	color: $m-gray;
-
-	// Reset to prevent conflict with other CSS files
- 	h1,
-	h2,
-	h3,
-	p,
-	a,
-	button,
-	table,
-	thead,
-	tr,
-	td,
-	button,
-	.toolbar {
-		background-color: transparent;
- 		color: $m-gray;
- 	}
-
-	// Buttons
-	button {
-		background-color: $t-dark;
-	}
-
-	// Tables
- 	table {
- 		strong {
-			color: $m-orange;
- 		}
- 		tbody tr {
- 			&:hover {
- 				background-color: $g-gray;
- 			}
- 			&.current {
- 				background-color: $m-orange;
-				td {
-					color: $t-dark;
-				}
-				&:hover td {
-	 				background-color: $g-red;
-					color: $t-light;
-	 			}
- 			}
- 		}
- 	}
-
-	// The toolbar
- 	.toolbar {
- 		background-color: $g-gray;
- 		@include box-shadow(0, 0, 4px, $g-gray);
-		img {
-			filter: brightness(0) invert(1);
-		}
- 	}
-
-	// Fixed top
-	&.fixed-top {
-		& .toolbar {
```

Comment:
Not 100% sure about removing the `&` here, but the css turn out same with and without as it is originally written as `& .toolbar` which should  be the same.